Some tiny fixes in recent typescript  typedefs changes

jymen opened the pull request:
The last commiting typedefs are breaking my code mainly around Synth stuff.
So here are the fixes for that 
I followed the documentation to provide argument semantics ... Hoping that there are no big changes compared to code
I made a test on my side with those changes and everything looks OK

I Attached the Typescript parsing  problem I got previous my changes in index.d.ts

<img width="676" alt="Screenshot 2021-05-07 at 11 46 05" src="https://user-images.githubusercontent.com/644653/117448703-5b084080-af3f-11eb-8683-8e2a3f2bd179.png">
<img width="886" alt="Screenshot 2021-05-07 at 11 14 31" src="https://user-images.githubusercontent.com/644653/117448786-73785b00-af3f-11eb-8fa7-ed0793b37701.png">

paulrosen commented:
Thanks for the update! I am starting to dive in to finish all the type definitions so I'll be doing a lot more changes to this. The more help I get to this the better!

I couldn't accept the pull request directly because it has a number of other changes - particularly it has an older version of package.json and deleted the package-lock.json.

Instead I cherry-picked the one commit that contained the type changes.

jymen commented:
> I couldn't accept the pull request directly because it has a number of other changes - particularly it has an older version of package.json and deleted the package-lock.json.
> 
Ok I’ll fix my repo then  ...


> On May 9, 2021 week 18, at 00:31, Paul Rosen ***@***.***> wrote:
> 
> 
> Thanks for the update! I am starting to dive in to finish all the type definitions so I'll be doing a lot more changes to this. The more help I get to this the better!
> 
> I couldn't accept the pull request directly because it has a number of other changes - particularly it has an older version of package.json and deleted the package-lock.json.
> 
> Instead I cherry-picked the one commit that contained the type changes.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/paulrosen/abcjs/pull/674#issuecomment-835548476>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAE5MLPYEMM3VPFFR7DL4WLTMW3TBANCNFSM44JX7C5Q>.
>
